TombEngine/Objects/TR3/Trap/turning_blade.cpp

```diff
@@ -0,0 +1,90 @@
+#include "framework.h"
+#include "Objects/TR3/Trap/turning_blade.h"
+
+#include "Game/animation.h"
+#include "Game/collision/collide_item.h"
+#include "Game/collision/collide_room.h"
+#include "Game/collision/Sphere.h"
+#include "Game/control/control.h"
+#include "Game/collision/Point.h"
+#include "Game/effects/effects.h"
+#include "Game/items.h"
+#include "Game/Lara/lara.h"
+#include "Game/Setup.h"
+#include "Sound/sound.h"
+#include "Specific/level.h"
+
+using namespace TEN::Collision::Sphere;
+using namespace TEN::Collision::Point;
+
+namespace TEN::Entities::Traps
+{
+	constexpr auto TURNING_BLADE_HARM_DAMAGE = 100;
+
+	const std::vector<unsigned int> TurningBladeHarmJoints = { 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+	const std::vector<Vector3i> TurningBladeBounds =
+	{
+		Vector3i(-CLICK(2), CLICK(2), -BLOCK(2)),
+		Vector3i(-896, -96, 96),
+		Vector3i(-CLICK(2), CLICK(2), -128),
+		Vector3i(0, -96, 96),
+		Vector3i(-CLICK(2), -384, -BLOCK(2)),
+		Vector3i(0, -96, 96),
+		Vector3i(384, CLICK(2), -BLOCK(2)),
+		Vector3i(0, -96, 96)
+	};
+
+	void InitializeTurningBlade(short itemNumber)
```

Comment:
If object does not require initialization, you can just omit initialization function and definition of it in setup code (tr3_objects.cpp). Empty function is not necessary.